docs/scripts.rst

```diff
@@ -130,6 +130,10 @@ Explanation for the keys:
 - ``urls``: dictionary of table names and the respective urls
 - ``version``: "1.0.0"
 
+``fields``: Although retriever automatically detects the schema of the table,
```

Comment:
I agree with these changes, and they might be more fitting if all of them (including the minor fields explanation) were put under the heading: **Full control over column names and data types**